tests/integration/cmd_link_unlink_test.go

```diff
@@ -41,14 +41,16 @@ var _ = Describe("odo link and unlink command tests", func() {
 	Context("when running help for link command", func() {
 		It("should display the help", func() {
 			appHelp := helper.CmdShouldPass("odo", "link", "-h")
-			Expect(appHelp).To(ContainSubstring("Link component to a service or component"))
+			// Check for -vmodule moduleSpec output which is in additional flags
+			Expect(appHelp).To(ContainSubstring("--vmodule moduleSpec"))
```

Comment:
Little confused with `--vmodule moduleSpec` part. Could you please help me understand why are we verifying the same part for both link and [unlink](https://github.com/openshift/odo/pull/3705/files#diff-93c99b01e2a566d8d27400ffb6e2ee51R53).

I think this is a common part which is reflected everywhere, I mean `odo push -h` will also have this. Correct me if I am wrong. I would suggest lets verify for specific substring for `odo link`

Reply:
Hey @prietyc123 we can use anything in the example. The only purpose of this test it to make sure that `--help` works. Since `--vmodule moduleSpec` is in pretty much every `--help` output. I used that instead.

Reply:
AFAIU here we are trying to get the examples/help for the `odo link` commands like `odo link <service> ...` . Wouldn't it be the case like somehow `odo link -h` gives you the default `odo -h` description/examples/help then even though the help message points to the wrong one it will pass, right?